core/src/main/scala/io/gearpump/util/Constants.scala

```diff
@@ -123,4 +123,8 @@ object Constants {
     classOf[HashPartitioner],
     classOf[ShuffleGroupingPartitioner],
     classOf[ShufflePartitioner])
+
+  //Security related
```

Comment:
I think one pair of authenticated principal and keytab is enough for all components.